docs/_sources/usage/default_values.md.txt

```diff
@@ -0,0 +1,65 @@
+```{toctree}
+:maxdepth: 2
+```
+
+# Default values
+
+Testing SQL queries can often involve repetitive setup for mock tables. In SQLMock, one effective way to streamline this process is by using default values. By setting reasonable defaults, you can significantly reduce the boilerplate code in your tests, especially when dealing with multiple input tables or complex queries. Let’s explore how you can efficiently implement this.
+
+## Utilizing Default Values in MockTable Fields
+
+Defining default values at the field level in your mock tables is straightforward. The default argument in the field definition allows you to set default values, which are particularly useful for ensuring that joins and other query functionalities operate correctly.
+
+Here's an example:
+
+```python 
+@table_meta(table_ref="data.users")
+class UserTable(BigQueryMockTable):
+    user_id = col.Int(default=1)
+    user_name = col.String(default="Mr. T")
+
+# Create instances of the UserTable with various combinations of defaults and specified values
+users = UserTable.from_dicts([
+    {}, # Uses default values --> {"user_id": 1, "user_name": "Mr. T"}
```

Comment:
**Thinking out loud:** Would it make sense to emphasize in the comment: 

> Left empty {} uses default values --> {"user_id": 1, "user_name": "Mr. T"}

Maybe you'll find better wording for it but I thought it might be useful to emphasize that this empty {} is the thing that pulls the defaults :)

Reply:
Yes I like the suggestion to make it more clear!